Sources/App/GithubService+IssueComment.swift

```diff
@@ -0,0 +1,89 @@
+import Vapor
+import Stevenson
+
+struct PingAction: Content {
+    let zen: String
+}
+
+struct CommentAction: Content {
+    let action: String
+    let comment: Comment
+    let issue: Issue
+
+    struct Comment: Content {
+        let body: String
+    }
+
+    struct Issue: Content {
+        let number: Int
+    }
+
+    struct PullRequest: Content {
+        let url: URL
+    }
+}
+
+extension GitHubService {
+    func webhook<T: Decodable>(from request: Request) throws -> Future<T> {
```

Comment:
Just to clarify that this method is decoding a received request (and not sending a webhook request or anything else)
```suggestion
    func decodeWebhook<T: Decodable>(from request: Request) throws -> Future<T> {
```